Up2dateService/Up2dateService/Service.cs

```diff
@@ -24,19 +24,23 @@ protected override void OnStart(string[] args)
         {
             const int clientStartRertyPeriodMs = 30000;
 
-            System.Diagnostics.Debugger.Launch(); //todo: remove!
```

Comment:
Probably it's better to wrap into 
`#ifdef DEBUG
#endif`
?

Reply:
I don't want it pops up even in DEBUG mode. So I uncomment is just in rare cases when it is really needed.

Reply:
In that case comment is ok